packages/angular_devkit/build_angular/src/tools/sass/rebasing-importer.ts

```diff
@@ -22,6 +23,38 @@ export interface DirectoryEntry {
   directories: Set<string>;
 }
 
+/**
+ * A prefix that is added to import and use directive specifiers that should be resolved
+ * as modules and that will contain added resolve directory information.
+ *
+ * This functionality is used to workaround the Sass limitation that it does not provide the
+ * importer file to custom resolution plugins.
+ */
+const MODULE_RESOLUTION_PREFIX = '__NG_PACKAGE__';
+
+function packModuleSpecifier(specifier: string, resolveDir: string): string {
```

Comment:
Suggestion: An interface for the unpacked format would allow you to accept/return the same type rather than disparite values.

```typescript
interface ModuleSpecifier {
  specifier: string;
  resolveDir?: string;
}
```